src/ontology/edits/oeo-model.omn

```diff
@@ -11,11 +10,11 @@ Prefix: xsd: <http://www.w3.org/2001/XMLSchema#>
 
 
 
-Ontology: <http://openenergy-platform.org/ontology/oeo-model/>
-<http://openenergy-platform.org/ontology/v0.0.1/oeo-model/>
-Import: <http://open-energy-ontology.org/ontology/imports/iao-annotation-module.owl>
-Import: <http://open-energy-ontology.org/ontology/imports/iao-module.owl>
-Import: <http://open-energy-ontology.org/ontology/imports/ro-module.owl>
+Ontology: <http://openenergy-platform.org/ontology/releases/oeo/oeo-model.omn>
```

Comment:
@jannahastings Here, I reference to the actual `.omn` file. The OEP does not handle oeo-models as an independent ontology - therefore https://openenergy-platform.org/ontology/oeo does not resolve.

Reply:
According the [W3](https://www.w3.org/Provider/Style/URI) we should probably change this to http://openenergy-platform.org/ontology/releases/oeo/oeo-model